build.sbt

```diff
@@ -1,48 +1,49 @@
 import org.scalajs.linker.interface.ModuleSplitStyle
-import sbtcrossproject.CrossPlugin.autoImport.{ crossProject, CrossType }
+import sbtcrossproject.CrossPlugin.autoImport.{crossProject, CrossType}
 
 val scala212 = "2.12.14"
 val scala213 = "2.13.7"
-val scala3   = "3.1.0"
+val scala3 = "3.1.0"
 val allScala = Seq(scala212, scala213, scala3)
 
-val akkaVersion            = "2.6.17"
-val catsEffect2Version     = "2.5.4"
-val catsEffect3Version     = "3.3.4"
-val circeVersion           = "0.14.1"
-val http4sVersion          = "0.23.7"
-val laminextVersion        = "0.14.2"
-val magnoliaVersion        = "0.17.0"
-val mercatorVersion        = "0.2.1"
-val playVersion            = "2.8.12"
-val playJsonVersion        = "2.9.2"
-val sttpVersion            = "3.3.18"
-val tapirVersion           = "0.19.3"
-val zioVersion             = "1.0.13"
+val akkaVersion = "2.6.17"
+val catsEffect2Version = "2.5.4"
+val catsEffect3Version = "3.3.4"
+val circeVersion = "0.14.1"
+val http4sVersion = "0.23.7"
+val laminextVersion = "0.14.2"
+val magnoliaVersion = "0.17.0"
+val mercatorVersion = "0.2.1"
+val playVersion = "2.8.12"
+val playJsonVersion = "2.9.2"
+val sttpVersion = "3.3.18"
+val scalafmtVersion = "3.1.2"
+val tapirVersion = "0.19.3"
+val zioVersion = "1.0.13"
 val zioInteropCats2Version = "2.5.1.0"
 val zioInteropCats3Version = "3.2.9.0"
-val zioConfigVersion       = "1.0.10"
-val zqueryVersion          = "0.2.10"
-val zioJsonVersion         = "0.1.5"
-val zioHttpVersion         = "1.0.0.0-RC17"
+val zioConfigVersion = "1.0.10"
+val zqueryVersion = "0.2.10"
+val zioJsonVersion = "0.1.5"
+val zioHttpVersion = "1.0.0.0-RC17"
```

Comment:
@pmeheut Please use the project formatter

Reply:
Done.

Reply:
Thanks